### PR TITLE
Header directory link

### PR DESCRIPTION
**Pull request by AdamJamesWalters**
🧐 What?

Provides a link to each folder in the path from within the Morty Pages.

🛠 How

Generating a link for each section of the path.

https://github.com/bbc/morty/issues/148
https://github.com/bbc/morty-docs/issues/31

### CODE REVIEW

**Comment by adamjc** [comment]
This looks great -- from a UX perspective (not that I really know anything...), I think it'd be good if the links appeared on every page, not just the index pages, and we could style it something like
Screenshot 2020-07-09 at 09 33 10 -- e.g. we're not removing the `morty-docs` header (which should probably change to a logo at some point)

(an example, not necessarily something to copy)

What do you think?

File: src/page-renderers/Components/Header.jsx [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
+  let headerLinks
+  let pathParts
+
+  if (relPath) {
+    pathParts = relPath.split('/')
+    headerLinks = getHeaderLinks(relPath, pathParts)
+  }
+
+  return (
+    <div className='container' style={Styles.navbar}>
```
Comment (TimPerry) [inline src/page-renderers/Components/Header.jsx:30]: Was just talking about accessibility on this with peeps. We probably want a `nav` > `ul` > `li` > `a` on these.

Comment (adamjc) [reply src/page-renderers/Components/Header.jsx:30]: For context: screen readers can parse `nav` elements as navigation elements

File: src/helpers/get-header-paths.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const getHeaderLinks = (relPath) => {
+  return (relPath || '')
+    .split('/')
+    .filter(part => !part || !part.includes('.'))
```
Comment (adamjc) [inline src/helpers/get-header-paths.js:4]: Do we strip out dotfiles/dotfolders? If we do not, does this filter mess up the expected behaviour?

e.g. say someone (for whatever reason) has stuff stored in `some-folder/.dotfolder/something.md`

**Comment by adamjc** [comment]
I know our testing of `morty-docs` isn't the greatest, but does this work with how _we_ use it? e.g. `domain/morty-docs/$repo`? Checking the root `.html` files appears to give a blank header:
Screenshot 2020-07-24 at 15 13 04, do we want this to have a link 'up', and would it when hosted how we host it?

**Pull request merged**